src/components/Spinner.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faSpinner } from '@fortawesome/free-solid-svg-icons'
+import { SizeProp } from '@fortawesome/fontawesome-svg-core'
+
+interface SpinnerProps {
+  spin?: boolean
+  style?: React.CSSProperties
+  size?: SizeProp
+}
+
+export const Spinner: React.FC<SpinnerProps> = ({ style, spin = true, size }) => (
+  <FontAwesomeIcon icon={faSpinner} style={style} spin={spin} size={size} />
```

Comment:
i would argue to make this more flexible is to allow icon to be a prop and set a default

Reply:
I just wanted to do the refactor. I think is better not to get ahead of ourselves. If you have an use case in mind, like have a  "🤘" spinner good, otherwise, I would think is better to add it when we need it